sdk/core/azure-core/inc/azure/core.hpp

```diff
@@ -14,6 +14,7 @@
 #include "azure/core/context.hpp"
 #include "azure/core/credentials.hpp"
 #include "azure/core/datetime.hpp"
+#include "azure/core/export.hpp"
```

Comment:
Any way to test the inclusion of this header within
https://github.com/Azure/azure-sdk-for-cpp/blob/a5f113e647710450870674e2270f75951f36914a/sdk/core/azure-core/test/ut/simplified_header.cpp#L16